cypress.config.ts

```diff
@@ -0,0 +1,50 @@
+import { defineConfig } from 'cypress';
+
+// @ts-expect-error -- no types
+import webpackConfig from './webpack.config.js';
```

Comment:
we could also define it as `.ts` file 🤔 
What do you think about naming it, so we know that it is for cypress? or inline it inside the cypress.config.ts